samples/notebooks/image-captioning-gcp/Image Captioning TF 2.0.ipynb

```diff
@@ -0,0 +1,893 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<h1> Image Captioning with Attention in Tensorflow 2.0 </h1>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook modifies the [Image Captioning with Attention Tensorflow 2.0 notebook](https://colab.sandbox.google.com/github/tensorflow/docs/blob/master/site/en/r2/tutorials/text/image_captioning.ipynb)\n",
+    "to work with kubeflow pipelines."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<h3> Download dataset and upload to GCS </h3>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First, we have to download the [MS COCO dataset](http://cocodataset.org/#download).  This sample uses both the 2014 train images and 2014 train/val annotations.  If you downloaded and extracted the dataset on your local system, you can upload it to GCS using `gsutil -m cp -r path/to/dataset/ gs://[YOUR-BUCKET-NAME]/ms-coco`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<h3> Setup project info and imports </h3>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Previously downloaded dataset and put onto GCS\n",
+    "GCS_DATASET_PATH = 'gs://[YOUR-BUCKET-NAME]/ms-coco'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Kubeflow project settings\n",
+    "EXPERIMENT_NAME = 'Image Captioning'\n",
+    "PROJECT_NAME = '[YOUR-PROJECT-NAME]' \n",
+    "PIPELINE_STORAGE_PATH = 'gs://[YOUR-BUCKET-NAME]/ms-coco/components' # path to save pipeline component images\n",
+    "BASE_IMAGE = 'gcr.io/intro-to-kubeflow-1/img-cap:latest' # using image created in README instructions"
+   ]
+  },
+  {
+   "cell_type": "code",
```

Comment:
Please add a cell to install the kfp, '!pip3 install kfp --upgrade', similar to https://github.com/kubeflow/pipelines/blob/master/samples/notebooks/DSL%20Static%20Type%20Checking.ipynb

Reply:
Done